terraform/templates/automate-install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+mkdir -p /etc/chef-automate
+mv /tmp/automate-config.toml /etc/chef-automate/config.toml
+
+/var/cache/marketplace/chef-automate deploy /etc/chef-automate/config.toml --accept-terms-and-mlsa --skip-preflight --airgap-bundle /var/cache/marketplace/chef-automate.bundle
```

Comment:
Don't we need to add the flag here to deploy chef-server?

Reply:
No that's done now in the automate config.toml,  Deployment team says that is more reliable

Reply:
I was not aware of that.  Thanks!